driverdataaggregation/main.py

```diff
@@ -93,11 +112,12 @@ def aggregation_scaling(self,
                                 None, # not setting threads per queue in throughput tests
                                 net_ifname,
                                 run_db_as_batch,
-                                db_hosts)
+                                db_hosts,
+                                wall_time)
 
 
-            for c_nodes, cpn, _bytes, t_per_dataset, c_threads in product(
-                client_nodes, clients_per_node, tensor_bytes, tensors_per_dataset, client_threads
+            for c_nodes, cpn, _bytes, t_per_dataset, c_threads, language in product(
+                client_nodes, clients_per_node, tensor_bytes, tensors_per_dataset, client_threads, languages
```

Comment:
It would probably be good to convert the language to lower-case here so that Fortran, FORTRAN, fortran are all accepted as well as cpp, CPP, Cpp.

Reply:
(apply this to the other tests as well)